src/test/java/org/ice4j/message/ChannelDataTest.java

```diff
@@ -0,0 +1,85 @@
+package org.ice4j.message;
+
+import org.ice4j.StunException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.ice4j.message.ChannelData.*;
+import static org.junit.Assert.*;
```

Comment:
I might be wrong but don't we tend to stray away from wildcarding these imports. It's certainly been mentioned by customers before but I'm not sure whether we have been doing this for PR's

Reply:
I've seen this in other PRs but also aware of some negative feedback so I'm inclined to change it.

Reply:
This is maybe more of a general discussion but I originally started importing Assert.* and ClassUnderTest.* to minimize the code and line length but I wonder if it is preferred to not do that and just deal with the longer lines?

Reply:
As discussed, I've removed the wildcard imports, called the test methods as ChannelData._method_ (which I think is clearer), added imports for each type of Assert and wrapped one line which then was > 80 chars.